setup.py

```diff
@@ -13,7 +13,8 @@
 
 
 install_requires = [
-    'lxml>=3.6.0',
+    'beautifulsoup4>=4.4.1',
+    'html5lib==0.999999',
```

Comment:
Should perhaps use the same version range as wagtail? https://github.com/torchbox/wagtail/blob/master/setup.py#L30